docs/tutorials/multi_grid.md

```diff
@@ -0,0 +1,46 @@
+# Tutorial 5: Use Multi-Grid From an Existed Training
+
+## Prerequisites
+1. First, your training should use step scheduler, since now multi-grid training doesn't support other scheduler type.
+2. Second, this training method is only used for 3D models such as I3D, SlowFast that use dense-sampling strategy. Models such as TSN, TSM is not supported.
+
+## Modify your config
+A basic reference can be the config for multi-grid trained [I3D](../../configs/recognition/i3d/i3d_r50_multigrid_32x2x1_100e_kinetics400_rgb.py). You may follow the steps to use multi-grid from a standard stepwise [config](../../configs/recognition/i3d/i3d_r50_32x2x1_100e_kinetics400_rgb.py).
```

Comment:
Considering the display for web docs. it should be `/configs/recognition/i3d/i3d_r50_multigrid_32x2x1_100e_kinetics400_rgb.py` and `/configs/recognition/i3d/i3d_r50_32x2x1_100e_kinetics400_rgb.py`